### PR TITLE
CHANGELOG for 2024.03

### PR DESCRIPTION
**Pull request by tobiasd**
Updating the CHANGELOG for the 2024.03 release.

This shall be the last PR merged into the `develop` branch before the release.

### CODE REVIEW

**Review by MrPetovan — APPROVED** [review]

File: CHANGELOG [hunk 1]
```diff
@@ -1,9 +1,53 @@
...
+    Improved the search performance [annando]
+    Improved spam detection [annando]
+    Improved the account overview on the moderation page [annando]
+    Improved account creation via CLI console [mexon]
+    Improved the Mastodon compatible API [annando, MrPetovan]
+    Improved logging of the system load value [annando]
+    Improved image handling [annando]
+    Fixed an issue that could lead to empty URLs in the server block list [annando]
+    Fixed XSS attacks [leoOliver, MrPetovan, snajafov]
+    Fixed enumberation attack on photo URLs [MrPetovan]
```
Comment (MrPetovan) [inline CHANGELOG]: This has been postponed to a future release.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: CHANGELOG [hunk 2]
```diff
@@ -1,9 +1,53 @@
...
+    Fixed an issue when importing emails [annando]
+    Fixed an issue that blocked users could still use the API [annando]
+    Fixed an issue when fetching remote content [anndndo, arcanicanis]
```
Comment (AlfredSK) [inline CHANGELOG]: > anndndo

Typo?

File: CHANGELOG (hunk 1 above)
Comment (tobiasd) [reply CHANGELOG]: Ok

**Comment by foss-** [comment]
Would it make sense to move the language changes to the very end? I'd think the release notes should start with big changes or highlights. Changes to the translations often aren't that exciting. Thanks for compiling the release notes.

Should we list the theme changes in greater detail? I think users are interested in such things:

```
Theme changes in frio: frio light blue is the new default theme, simplify theme settings, macOS emoji picker fixed, fixed several UI bugs, removed legacy schemes [annando, foss-, haheute, MrPetovan, Raroun, teddy15]
Theme changes in vier: fixed posts to a group [annando]
```

For the news post I am wondering if the "how to update" section is really needed. Imo that should be in the documentation and the release announcement should focus on the changes in the release. That would make for a more concise news post. Thoughts?

**Comment by tobiasd** [comment]
Translation update and documentation work *are* the most important updates ;-)

And the release announcement is not written down in the CHANGELOG file.

File: CHANGELOG [hunk 3]
```diff
@@ -1,9 +1,52 @@
...
+    pnut:
+      Connector addon was added [spacenerdmo]
+    tesseract
+      Added the addon to generate image descriptions from images via OCR [annando]
+    url_replace
+      Added addon to replace URLs from Twitter, Youtube and some others using 12ft.io [toddy15]
 
   Closed Issues
+      903,  7732,  8768, 11142, 13220, 13293, 13765, 13768, 13809,
+    13814, 13814, 13818, 13819, 13822, 13823, 13828, 13839, 13844,
```
Comment (Raroun) [inline CHANGELOG]: Issue #13837 was also fixed :)

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**